Source/popup.html

```diff
@@ -1,23 +1,19 @@
-<!doctype html>
+<!DOCTYPE html>
 <html ng-app="PopupApp">
   <head>
     <title>Mind The Word</title>
-
     <link href="assets/css/bootstrap.css" rel="stylesheet">
-
-    <script type="text/javascript" src="assets/js/angular.min.js"></script>
+    <script type="text/javascript" src="assets/js/angular.min.js"></script> //angularjs lib
```

Comment:
@Ceilican , the //angularjs lib will get displayed in popup. This is wrong syntax for a HTML comment